syft/execution/placeholder.py

```diff
@@ -146,8 +146,19 @@ def copy(self):
         copy operations happen locally where we want to keep reference to the same
         instantiated object. As the child doesn't get sent, this is not an issue.
         """
-        placeholder = PlaceHolder(tags=self.tags, owner=self.owner, shape=self.expected_shape)
+        placeholder = PlaceHolder(
+            role=self.role,
+            tracing=self.tracing,
+            tags=self.tags,
+            owner=self.owner,
+            shape=self.expected_shape,
+        )
         placeholder.child = self.child
+
+        if self.tracing:
+            command = ("copy", self, tuple(), {}), placeholder
+            self.role.register_action(command, syft.execution.computation.ComputationAction)
+
```

Comment:
Hadn't considered that we might want to do `copy`ing inside `Plans`!

Reply:
I recall you mentioned that you don't want to capture `send` action in `placeholder.send`, but rather in same place as other commands are captured.
Perhaps we should try doing same for copy?

Reply:
I think it's fine for now.